### PR TITLE
feat: support to create commit comment in GitHub

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: scm/driver/github/testdata/commit_comment_response.json.goden [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "Link": "https://github.com/octocat/Hello-World/commit/6dcb09b5b57875f334f61aebed695e2e4193db5e#commitcomment-1",
+    "Author": {
+        "Login": "octocat",
+        "Name": "",
+        "Email": "",
+        "Avatar": "https://github.com/images/error/octocat_happy.gif"
+    },
+    "Created": "2011-04-14T16:00:49Z",
+    "Updated": "2011-04-14T16:00:49Z"
+}
```
Comment (ankitm123) [inline scm/driver/github/testdata/commit_comment_response.json.goden:16]: NIT: file name is golden, and EOL

Comment (LinuxSuRen) [reply scm/driver/github/testdata/commit_comment_response.json.goden]: Yes. Of course.

**Review by ankitm123 — COMMENTED** [review]
Minor comments, but otherwise looks good to me, thanks for adding tests :)

File: scm/driver/github/testdata/commit_comment_response.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (ankitm123) [inline scm/driver/github/testdata/commit_comment_response.json:1]: golden file?

**Comment by msvticket** [comment]
PR is abandonded. Author doesn't responded to requests.

/close

**Pull request closed without merging**